config/maven/updateRules.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
To be consistent with Maven conventions I think this file should be located at `src\main\resources`.

Reply:
done